### ISSUE TITLE
Increased circulating prolactin concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4849]
**Preferred term label:**

Increased circulating prolactin concentration
**Synonyms**
Hyperprolactinemia

**Definition (free text, please give PubMed ID)**
An elevated concentration of prolactin in the blood circulation

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MEN1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0000870

**Comment by pnrobinson** [comment]
done
